front/src/infoboxes/Infoboxes.css

```diff
@@ -0,0 +1,94 @@
+.exclamation {
+  position: absolute;
+  bottom: 1px;
+  height: 20px;
+  width: 20px;
+  margin-right: 16px;
+  margin-bottom: 0px;
+  clear: both;
+  min-height: 22px;
+  min-width: 22px;
+}
+
+.info-flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-bottom: 32px;
+}
+
+.info-item {
+  display: flex;
+  min-width: 320px;
+  width: 30vw;
+  height: 38px;
+  justify-content: left;
+}
+
+.box {
+  float: left;
+  height: 20px;
+  width: 20px;
+  margin-right: 8px;
+  margin-left: 0px;
+  margin-bottom: 0px;
+  border: 1px solid black;
+  clear: both;
+  min-height: 20px;
+  min-width: 20px;
+}
+
+#open-info {
+  display: flex;
+  background: #658f60;
+}
+
+#valvoja-info {
```

Comment:
Not necessary but it's probably best to only use English in the code in case of English group in the future

Reply:
Changed all the finnish info that i could find into english